node/resolvers/auth/index.ts

```diff
@@ -15,7 +15,7 @@ const makeRequest = async (ctx, url) => {
 
 export const mutations = {
   sendEmailVerification: async (_, args, { vtex: ioContext }) => {
-    const { data: { authenticationToken } } = await makeRequest(ioContext, paths.getTemporaryToken())
+    const { data: { authenticationToken } } = await makeRequest(ioContext, paths.getTemporaryToken(ioContext.account, ioContext.account))
     await makeRequest(ioContext, paths.sendEmailVerification(args.email, authenticationToken))
     return { authToken: authenticationToken }
```

Comment:
A thing I did in the VTEX ID Admin APP was to pass down this `authorizationToken` as a cookie, instead of passing the value. Either way works, but the cookie looks a little bit better.

Also, this token identifies the login session. Renaming it could make future maintenance easier.
E.g.: `session`, `sessionId`, `loginSessionId`, `temporarySession`, ...